src/main/java/uk/gov/ch/oauth/Oauth2.java

```diff
@@ -241,4 +245,22 @@ private void removeZXSInfo(Map<String, Object> sessionData) {
             sessionFactory.getDefaultStore().delete(zxsKey);
         }
     }
+
+    public String getOriginalRequestURL(final HttpServletRequest request) {
```

Comment:
Can we reduce visibility of method please

Reply:
The method is now private's and the tests refactored accordingly.